.github/workflows/lint-autoformat.yml

```
@@ -0,0 +1,64 @@
+name: Apply lint suggestions
+
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened]
+    branches: [main]
+
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches: [gh/**/base]
+
+jobs:
+  # NB: This doesn't work with forked PR https://github.com/parkerbxyz/suggest-changes/issues/33 :(
+  checkout-pytorch-grab-lintrunner:
+    permissions:
```

This looks easier and safer

```suggestion
    permissions: read-all
```